MyHeartCounts/Task Handling/PerformTaskAnchor.swift

```diff
@@ -0,0 +1,252 @@
+//
+// This source file is part of the My Heart Counts iOS application based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2025 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+// swiftlint:disable all
+
+// TODO terminology here!!! (perform vs present vs active! task vs action!)
```

Comment:
I like the perform wording here; indicates a longer running process compared to present/activate.

Reply:
Sound good; thank you!